adafruit_bitmapsaver.py

```diff
@@ -177,7 +180,7 @@ def save_pixels(
             raise ValueError(
                 "Third argument must be a Palette or ColorConverter for a Bitmap save"
             )
-    elif not isinstance(pixel_source, Display):
+    elif not isinstance(pixel_source, BusDisplay):
```

Comment:
```suggestion
    elif not isinstance(pixel_source, (BusDisplay, FramebufferDisplay)):
```
Please add `FramebufferDisplay` here as well, it has`fill_row()` and is supported by this library too.

It will need to import it as well
```
from framebufferio import FramebufferDisplay
```